h2o-bindings/bin/gen_python.py

```diff
@@ -251,25 +253,26 @@ def extend_schema_params(param):
             property_doc = "Type: ``%s``" % dtype
         property_doc += ("." if pdefault is None else "  (default: ``%s``)." % stringify(pdefault))
 
-        deprecated = pname in get_customizations_for(algo, 'deprecated', [])
+        deprecated = pname in get_customizations_or_defaults_for(algo, 'deprecated', [])
         yield "    @property"
         yield "    def %s(self):" % pname
         yield '        """'
         yield bi.wrap("%s%s" % ("[Deprecated] " if deprecated else "", param.get('help')), indent=8*' ')  # we need to wrap only for text coming from server
         yield ""
-        yield bi.wrap(property_doc, indent=8*' ')
-        custom_property_doc = get_customizations_for(algo, "doc.{}".format(pname))
+        custom_property_doc = get_customizations_or_defaults_for(algo, "doc.{}".format(pname))
         if custom_property_doc:
-            yield ""
             yield reformat_block(custom_property_doc, 8)
-        property_examples = get_customizations_for(algo, "examples.{}".format(pname))
+        else:
+            yield bi.wrap(property_doc, indent=8*' ')
```

Comment:
@sebhrusen this is a way how we can replace property_doc with custom_property_doc. Once we discuss what API will be given to a user, it will become clearer whether we need these hooks or not. At least there is a way to hook docs as well.